### PR TITLE
Updating CSP for icons

### PR DESCRIPTION
**Pull request by adityaagashe1**

### CODE REVIEW

File: etc/csp_whitelist.xml [hunk 1]
```diff
@@ -16,5 +16,13 @@
                 <value id="extend-apis" type="host">https://*.extend.com/</value>
             </values>
         </policy>
+        <policy id="image-src">
+            <values>
+                <value id="extend-platformsandbox-images" type="host">https://shipping-offers-static-images-bucket-platformsandbox.s3.amazonaws.com/*</value>
+                <value id="extend-stage-images" type="host">https://shipping-offers-static-images-bucket-stage.s3.amazonaws.com/*</value>
+                <value id="extend-prod-images" type="host">https://shipping-offers-static-images-bucket-prod.s3.amazonaws.com/*</value>
+                <value id="extend-dev-images" type="host">https://shipping-offers-static-images-bucket-dev.s3.amazonaws.com/*</value>
```
Comment (billhefty) [inline etc/csp_whitelist.xml]: Demo?

Comment (adityaagashe1) [reply etc/csp_whitelist.xml]: Added!

**Review by scott-taylor-extend — COMMENTED** [review]
You don't need https?

**Review by scott-taylor-extend — APPROVED** [review]

**Comment by adityaagashe1** [comment]
> You don't need https?

We don't need https. I've verified it.

**Review by billhefty — APPROVED** [review]

**Pull request merged**